buildSrc/src/main/java/Dependencies.kt

```diff
@@ -281,14 +281,6 @@ object RepoMatching {
     const val mozilla = "org\\.mozilla\\..*"
     const val androidx = "androidx\\..*"
     const val comAndroid = "com\\.android.*"
-    const val comGoogleFirebase = "com\\.google\\.firebase"
-    const val comGoogleTesting = "com\\.google\\.testing\\..*"
-
-    /**
-     * A matcher for com.google.android.* with one exception: the espresso-contrib dependency includes the
-     * accessibility-test-framework dependency, which is not available in the google repo. As such, we must
-     * explicitly exclude it from this regex so it can be found on Maven Central. Note that the transitive
-     * dependency com.google.guava is also not available on google's repo.
-     */
-    const val comGoogleAndroid = "com\\.google\\.android\\..*"
```

Comment:
Since you are removing `comGoogleAndroid` here, you have to remove the references to it in the other files.

Reply:
I removed all the references of comGoogleAndroid and now it is not found in any other files

![image](https://user-images.githubusercontent.com/22738464/197960141-6c6f9c9e-327c-4b40-9d79-42713dfc3c90.png)